prometheus-exporters-formula/prometheus-exporters-formula.spec

```diff
@@ -18,13 +18,15 @@
 %define fname prometheus-exporters
 %define fdir  %{_datadir}/susemanager/formulas
 Name:           prometheus-exporters-formula
-Version:        0.9.0
+Version:        0.9.1
 Release:        0
 Summary:        Salt formula for installing and configuring Prometheus exporters
 License:        Apache-2.0
 Group:          System/Packages
 Url:            https://github.com/SUSE/salt-formulas
 Source:         prometheus-exporters-formula-%{version}.tar.gz
+BuildRequires:  python3
+Requires:       python3
```

Comment:
One minor thing already commented on the other PR: do we really need a `BuildRequires` here?

Reply:
Yes, IBS tests fail without.

Reply:
Strange, where did you run the test exactly? Can you give us a URL?

Reply:
https://build.suse.de/package/live_build_log/home:witekbedyk:branches:Devel:Galaxy:Manager:Head/prometheus-exporters-formula/SLE_15_SP3/x86_64

Reply:
Ok, it's because of the new `prometheus-exporters-formula/scripts/migrate_formula_data.py`, which now you are running as part of `post`.

While the "BuildRequires" is not correct (because `Post` uses `Requires`), this is required to make the tests happy.

Go ahead and merge.